assets/dev/js/editor/elements/views/column.js

```diff
@@ -132,6 +132,10 @@ ColumnView = BaseElementView.extend( {
 
```

Comment:
@matipojo, maybe we need to add a task `isDroppingAllowed` to use `isValidChild`